nilearn/decomposition/canica.py

```diff
@@ -185,4 +185,10 @@ def fit(self, imgs, y=None, confounds=None):
             ica_maps[abs_ica_maps < threshold] = 0.
         self.components_ = ica_maps
 
+        # BF for issue #570: flip signs in each component to have more +ve than
+        # -ve voxels
```

Comment:
I don't think this comment is accurate anymore now that we use the max heuristics.

Also I'd rather have a comment that explain that it is a heuristics for reproducible maps rather than mentioning the GH issue.